grapher/barCharts/DiscreteBarChart.test.ts

```diff
@@ -111,11 +111,11 @@ describe("barcharts with columns as the series", () => {
             },
         })
         expect(chart.formatValue(chart.series[0])).toEqual({
-            valueString: "1,002",
+            valueString: "1k",
```

Comment:
I was just curiously scrolling around, can ignore if this is not yet ready, but we very likely **don't want to replace `1,002` with `1k` here**. 

And in general `k` should not be used for thousands unless we are opting for short number prefixes, which is really a proxy for "make this number take up as little space as possible" which could be a separate setting from the number prefixes, although it aligns pretty well with it so far that it may not be necessary.